src/components/TokenImg.tsx

```diff
@@ -1,15 +1,21 @@
 import { Currency } from '@uniswap/sdk-core'
-import missingTokenSrc from 'assets/missing-token-image.png'
 import { useToken } from 'hooks/useCurrency'
 import useCurrencyLogoURIs from 'hooks/useCurrencyLogoURIs'
 import { useCallback, useMemo, useState } from 'react'
 import styled from 'styled-components/macro'
 
 const badSrcs = new Set<string>()
 
-const MissingTokenImg = styled.img`
-  height: 1em;
-  width: 1em;
+const MissingTokenImg = styled.div<{ fontSize?: number; circleSize?: number }>`
+  background-color: ${({ theme }) => theme.container};
+  border-radius: 100%;
+  color: ${({ theme }) => theme.primary};
+  font-size: ${({ fontSize }) => fontSize || 9}px;
```

Comment:
The font specification should be done by nesting a proper type from `src/theme/type.tsx` under the `MissingTokenImg`.
Also, this is no longer an img, it is a div, so it can just be named `MissingToken` (or be made a proper functional element).

Reply:
i'm not sure that specifying a new font makes sense in this case since that font is imo very unlikely to be used outside of the context of this specific image? and semantically this element is playing the role of an image - it's an implementation detail to me that it's a div. what do you mean proper functional element?